packages/components/form-control/tests/form-control.test.tsx

```diff
@@ -59,73 +61,56 @@ describe("<FormControl />", () => {
         </FormControl>
       </>,
     )
-    expect(
-      screen
-        .getByTestId("FormControl-a")
-        .getElementsByClassName("ui-form__error-message")[0],
-    ).toHaveTextContent("Email is required.")
-    expect(
-      screen
-        .getByTestId("FormControl-b")
-        .getElementsByClassName("ui-form__helper-message")[0],
-    ).toHaveTextContent("Please enter your email")
+    expect(screen.queryByText("Email is required.")).toBeInTheDocument()
```

Comment:
@illionillion 

You should avoid to use `.queryBy*`, because the only purpose to use it is checking for non-existance.
Please see: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence

Use `getBy*` or `findBy*`.